charts/wildfly/values.schema.json

```diff
@@ -593,6 +593,38 @@
                     }
                   }
                 },
+                "commandOverride": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Override the default command for the WildFly container."
+                },
+                "argsOverride": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Override the default args for the WildFly container."
+                },
+                "securityContext": {
```

Comment:
I think we should keep the security context as a free-form object (without specifying its `properties`) so that all its [fields](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#podsecuritycontext-v1-core) could be specified. 
We are already doing that some fields such as [`deploy.resources`](https://github.com/wildfly/wildfly-charts/blob/0e8d57c617d05e3b96a71f4afacdf17976543953/charts/wildfly/values.schema.json#L316)